lib/ansible/module_utils/identity/keycloak/keycloak.py

```diff
@@ -86,9 +86,9 @@ def get_token(base_url, validate_certs, auth_realm, client_id,
     payload = dict(
         (k, v) for k, v in temp_payload.items() if v is not None)
     try:
-        r = json.load(open_url(auth_url, method='POST',
+        r = json.loads(open_url(auth_url, method='POST',
                                validate_certs=validate_certs,
-                               data=urlencode(payload)))
+                               data=urlencode(payload)).read().decode('utf-8'))
```

Comment:
Hey, could you please use `to_native` instead of `decode`? You can find its implementation [here](https://github.com/ansible/ansible/blob/6f04f87a139c0ed844f56b4d9e70f71c096f4a66/lib/ansible/module_utils/_text.py#L276) and example usages [here](https://github.com/ansible/ansible/blob/6f04f87a139c0ed844f56b4d9e70f71c096f4a66/lib/ansible/modules/web_infrastructure/jenkins_plugin.py#L318)